router/core/subgraph_transport.go

```diff
@@ -43,22 +42,8 @@ func (tt *TimeoutTransport) RoundTrip(req *http.Request) (*http.Response, error)
 	subgraph := rq.ActiveSubgraph(req)
```

Comment:
This will also fix the case when timeouts has been defined for all sugraphs right? Please add a test that covers it.

Reply:
Added